<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>物联网设备管理平台 2.0</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>750</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>700</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="DataButton" name="pushButton_led1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_led2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_beep">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_hum">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>270</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_relay">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_temp">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_vol">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>420</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_cpu">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_vr">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>420</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_cur">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>560</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="DataButton" name="pushButton_pw">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>560</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>341</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>69</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataButton</class>
   <extends>QPushButton</extends>
   <header>databutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
